pkg/controller/clusters/clusterapi.go

```diff
@@ -37,3 +38,11 @@ func CheckControlPlaneReady(ctx context.Context, client client.Client, log logr.
 	log.Info("CAPI control plane is ready")
 	return controller.Result{}, nil
 }
+
+func CapiClusterObjectKey(cluster *anywherev1.Cluster) client.ObjectKey {
```

Comment:
I would maybe move this to `pkg/controller/clusterapi.go`
It seems something closer to what we have there, I think.

Reply:
Makes sense. I was on the fence on which file to use so I definitely see your point, updated it!